### PR TITLE
fixed URLs and fixed ray download error

### PR DESCRIPTION
**Pull request by sujee**
## Why are these changes needed?

- fixed image and colab urls to point to main repo
- fixed model download error in pdf2parquet in ray version

### CODE REVIEW

**Comment by shahrokhDaijavad** [comment]
@sujee I tested this today. The local versions of both Python and Ray ran successfully. The Python version on Colab is fine, but the Ray version on Colab has the following problem:
Coming out of the pdf2parquet in Step 3.2 and inspecting the output in Step 3.3, I can only see mars.pdf and earth.pdf is missing! Does it mean that earth.pdf was not converted to pq? Please look into this.

**Comment by shahrokhDaijavad** [comment]
@sujee Thanks for your help in debugging the problem! If I click on "Open with Colab" from the notebook in the repo, it takes the old version of the the notebook from the dev branch. In order to test the new version, I had to open Sujee's fork directly from the Colab  [https://colab.research.google.com/github/sujee/data-prep-kit/blob/intro-example2/examples/notebooks/intro/dpk_intro_1_ray.ipynb] and then I was able to run the Ray version in Colab successfully.

**Review by shahrokhDaijavad — APPROVED** [review]
LGTM. I have tested both notebooks locally and on Google Colab.

File: examples/notebooks/intro/dpk_intro_1_python.ipynb [hunk 1]
```diff
@@ -118,9 +118,9 @@
    "source": [
     "if RUNNING_IN_COLAB:\n",
     "    !mkdir -p 'input/solar-system'\n",
-    "    !wget -O 'input/solar-system/earth.pdf'  'https://raw.githubusercontent.com/sujee/data-prep-kit/intro-example1/examples/notebooks/intro/input/solar-system/earth.pdf'\n",
-    "    !wget -O 'input/solar-system/mars.pdf'  'https://raw.githubusercontent.com/sujee/data-prep-kit/intro-example1/examples/notebooks/intro/input/solar-system/mars.pdf'\n",
-    "    !wget -O 'my_utils.py'  'https://raw.githubusercontent.com/sujee/data-prep-kit/intro-example1/examples/notebooks/intro/my_utils.py'"
+    "    !wget -O 'input/solar-system/earth.pdf'  'https://raw.githubusercontent.com/IBM/data-prep-kit/dev/examples/notebooks/intro/input/solar-system/earth.pdf'\n",
```
Comment (touma-I) [inline examples/notebooks/intro/dpk_intro_1_python.ipynb:121]: How were these files added ? Can anyone use this as a repo for storing/sharing documents ?

File: examples/notebooks/intro/dpk_intro_1_python.ipynb [hunk 2]
```diff
@@ -118,9 +118,9 @@
...
+    "    !wget -O 'input/solar-system/mars.pdf'  'https://raw.githubusercontent.com/IBM/data-prep-kit/dev/examples/notebooks/intro/input/solar-system/mars.pdf'\n",
+    "    !wget -O 'my_utils.py'  'https://raw.githubusercontent.com/IBM/data-prep-kit/dev/examples/notebooks/intro/my_utils.py'"
```
Comment (touma-I) [inline examples/notebooks/intro/dpk_intro_1_python.ipynb:123]: Why is this here? Why isn't that in the git repo? What am I missing ?

File: examples/notebooks/intro/dpk_intro_1_ray.ipynb [hunk 3]
```diff
@@ -504,7 +504,8 @@
         "    \"data_files_to_use\": ast.literal_eval(\"['.pdf']\"),\n",
         "    # orchestrator\n",
         "    \"runtime_worker_options\": ParamsUtils.convert_to_ast(worker_options),\n",
-        "    \"runtime_num_workers\": MY_CONFIG.RAY_RUNTIME_WORKERS,\n",
+        "    # \"runtime_num_workers\": MY_CONFIG.RAY_RUNTIME_WORKERS,\n",
```
Comment (touma-I) [inline examples/notebooks/intro/dpk_intro_1_ray.ipynb]: Please decide on using 1 or MY_CONFIG.RAY_RUNTIME_WORKERS and remove commented line. Thanks

**Review by touma-I — COMMENTED** [review]
Please review comment and decide to act on/ or ignore comments . Thanks.

File: examples/notebooks/intro/dpk_intro_1_python.ipynb (hunk 2 above)
Comment (shahrokhDaijavad) [reply examples/notebooks/intro/dpk_intro_1_python.ipynb:123]: The .py file is in the repo, but to run on Google Colab, Sujee puts it in raw.githubusercontent.com. @sujee Can we take this file from the repo itself, as: https://github.com/IBM/data-prep-kit/blob/dev/examples/notebooks/intro/my_utils.py?

Comment (sujee) [reply examples/notebooks/intro/dpk_intro_1_python.ipynb:123]: yes, on colab, we need to download any files besides the notebook.
These are being downloaded from the official repo location
